lib/controller/handlers/callback.js

```diff
@@ -40,11 +41,26 @@ module.exports = {
         const {
           pre: {
             user,
-            login
+            login,
+            postResponseValidationErrorHandler
           }
         } = request
 
-        return Boolean(await login(request, user.nameID, user))
+        const loginResult = await login(request, user.nameID, user)
+        if (isObject(loginResult) && !loginResult.success) {
```

Comment:
Unit tests are expected to be added to test all of these flows (see coverage is at 100%) for this one

Reply:
I added tests.